src/coreclr/tools/aot/ObjWriter/objwriter.cpp

```diff
@@ -615,9 +615,23 @@ CVRegNum ObjectWriter::GetCVRegNum(ICorDebugInfo::RegNum RegNum) {
     }
     break;
   case Triple::x86_64:
-    if (Amd64::ICorDebugInfo::REGNUM_RAX <= RegNum &&
-        RegNum <= Amd64::ICorDebugInfo::REGNUM_R15) {
-      return RegNum - Amd64::ICorDebugInfo::REGNUM_RAX + CV_AMD64_RAX;
+    switch ((Amd64::ICorDebugInfo::RegNum)RegNum) {
+      case Amd64::ICorDebugInfo::REGNUM_RAX: return CV_AMD64_RAX;
```

Comment:
Let's use consistent indentation style. In LLVM codebase they do not indent `case`s within a `switch`. I think they also enforce having the `default` label, producing a warning otherwise.

Reply:
See build output in this PR:
```
  D:\workspace\_work\1\s\src\coreclr\tools\aot\ObjWriter\objwriter.cpp(635,5): warning C4062: enumerator 'Amd64::ICorDebugInfo::REGNUM_COUNT' in switch of enum 'Amd64::ICorDebugInfo::RegNum' is not handled [D:\workspace\_work\1\s\artifacts\llvm\build\tools\ObjWriter\objwriter.vcxproj]
  D:\workspace\_work\1\s\src\coreclr\inc\cordebuginfo.h(69): message : see declaration of 'Amd64::ICorDebugInfo::RegNum' [D:\workspace\_work\1\s\artifacts\llvm\build\tools\ObjWriter\objwriter.vcxproj]
  D:\workspace\_work\1\s\src\coreclr\tools\aot\ObjWriter\objwriter.cpp(635,5): warning C4062: enumerator 'Amd64::ICorDebugInfo::REGNUM_AMBIENT_SP' in switch of enum 'Amd64::ICorDebugInfo::RegNum' is not handled [D:\workspace\_work\1\s\artifacts\llvm\build\tools\ObjWriter\objwriter.vcxproj]
  D:\workspace\_work\1\s\src\coreclr\inc\cordebuginfo.h(69): message : see declaration of 'Amd64::ICorDebugInfo::RegNum' [D:\workspace\_work\1\s\artifacts\llvm\build\tools\ObjWriter\objwriter.vcxproj]
```